app-config.yaml

```diff
@@ -65,76 +71,146 @@ proxy:
-
-jenkins:
-  instances:
-    - name: cft
-      baseUrl: http://localhost:8080
-      username: admin
-      apiKey: ${JENKINS_BASIC_AUTH_HEADER}
-
-auth:
-  # see https://backstage.io/docs/auth/ to learn about auth providers
-  environment: development
-  providers:
-    github:
-      development:
-        clientId: ${AUTH_GITHUB_CLIENT_ID}
-        clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
-    microsoft:
-      development:
-        clientId: ${AUTH_MICROSOFT_CLIENT_ID}
-        clientSecret: ${AUTH_MICROSOFT_CLIENT_SECRET}
-        tenantId: ${AUTH_MICROSOFT_TENANT_ID}
+    type: 'local' # Alternatives - 'googleGcs' or 'awsS3' or 'azureBlobStorage' or 'openStackSwift'. Read documentation for using alternatives.
 
-scaffolder:
+integrations:
   github:
-    token: ${GITHUB_TOKEN}
-    visibility: public # or 'internal' or 'private'
+    - host: github.com
+      token: ${GITHUB_TOKEN}
 
 catalog:
+  import:
+    entityFilename: catalog-info.yaml
+    pullRequestBranchName: backstage-integration
   rules:
-    - allow: [Component, System, API, Group, User, Resource, Location, Template]
+    - allow:
+        - Component
+        - API
+        - Resource
+        - Group
+        - User
+        - Template
+        - System
+        - Domain
+        - Location
+
   processors:
     microsoftGraphOrg:
       providers:
         - target: https://graph.microsoft.com/v1.0
           authority: https://login.microsoftonline.com
-          # If you don't know you tenantId, you can use Microsoft Graph Explorer
-          # to query it
-          tenantId: ${AUTH_MICROSOFT_TENANT_ID}
-          # Client Id and Secret can be created under Certificates & secrets in
-          # the App registration in the Microsoft Azure Portal.
-          clientId: ${AUTH_MICROSOFT_CLIENT_ID}
-          clientSecret: ${AUTH_MICROSOFT_CLIENT_SECRET}
-          # Optional filter for user, see Microsoft Graph API for the syntax
-          # See https://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0#properties
-          # and for the syntax https://docs.microsoft.com/en-us/graph/query-parameters#filter-parameter
-          # This and userGroupMemberFilter are mutually exclusive, only one can be specified
-          userFilter: accountEnabled eq true
-          # Optional filter for users, use group membership to get users.
-          # This and userFilter are mutually exclusive, only one can be specified
-          #userGroupMemberFilter: "displayName eq 'Backstage Users'"
-          # Optional filter for group, see Microsoft Graph API for the syntax
-          # See https://docs.microsoft.com/en-us/graph/api/resources/group?view=graph-rest-1.0#properties
-          groupFilter: securityEnabled eq true
+          tenantId: ${MICROSOFT_GRAPH_TENANT_ID}
+          clientId: ${MICROSOFT_GRAPH_CLIENT_ID}
+          clientSecret: ${MICROSOFT_GRAPH_CLIENT_SECRET_TOKEN}
+          userFilter: accountEnabled eq true and userType eq 'member'
```

Comment:
can you revert the userFilter and groupFilter changes please?

justice users are guests so we can't filter on member only

Reply:
Done